src/lib/init-hooks/core-hooks/http-logger.js

```diff
@@ -0,0 +1,50 @@
+const url = require('url');
+
+module.exports = function(BoringInjections) {
+
+  const {
+    boring,
+    logger,
+    config,
+  } = BoringInjections;
+
+  boring.before('add-hooks', function() {
+
+    boring.before('app.use', function(ctx) {
+      if (ctx.name === 'context') {
+        boring.app.use('http-logger', function loggerMiddleware(req, res, next) {
+
+          const parsedUrl = url.parse(req.originalUrl);
+          const path = (parsedUrl.pathname || '').toLowerCase();
+          const ext = path.split('.').pop();
+          const logLevel = (path.indexOf('_hmr')>0 ||
+                            ext.endsWith('map') ||
+                            ext.endsWith('js') ||
+                            ext.endsWith('ico') ||
+                            ext.endsWith('css') ||
+                            ext.endsWith('jpg') ||
+                            ext.endsWith('jpeg') ||
+                            ext.endsWith('png') ||
+                            ext.endsWith('gif')) ? 'trace' : 'info';
+
+          const objToLog = {
+            path,
+            method: req.method,
+          };
+
+          if (config.get('boring.logUserAgent', true)) {
+            objToLog.userAgent = req.headers['user-agent'];
```

Comment:
No need to see userAgent in local dev.  This value is by default to true _except_ in local host dev is `false`